src/style.css

```diff
@@ -1,3 +1,76 @@
+:root {
+  --cyan: #00acdc
```

Comment:
use more meaningful names
```suggestion
  --accent-color: #00acdc
```